osquery/tables/applications/posix/prometheus_metrics.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+#include <sstream>
+
+#include <osquery/config.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include <osquery/core/conversions.h>
+
+#include <osquery/tables/applications/posix/prometheus_metrics.h>
+#include <osquery/tables/applications/posix/prometheus_metrics_utils.h>
+
+namespace http = boost::network::http;
+
+namespace osquery {
+namespace tables {
+
+QueryData& parseScrapeResults(std::map<std::string, retData>& scrapeResults,
+                              QueryData& rows) {
+  for (auto const& target : scrapeResults) {
+    std::stringstream ss(target.second.content);
+    std::string dest;
+    std::string ts(std::to_string(target.second.timestampMS.count()));
+
+    while (std::getline(ss, dest)) {
+      if (!dest.empty() && dest[0] != '#') {
+        std::vector<std::string> metric(osquery::split(dest, " "));
+
+        if (metric.size() > 1) {
```

Comment:
yeah, if there's less than that number of elements, we don't have a full metric pair.  In the wild there should always be 2.  I used "> 1" just in case there's some unexpected character (i.e '\t' or something)